console/backend/src/main/java/org/frankframework/management/web/ConfigurationStatus.java

```diff
@@ -0,0 +1,196 @@
+			builder.addHeader(BusMessageUtils.HEADER_ADAPTER_NAME_KEY, "*ALL*");
+			callAsyncGateway(builder);
+		} else {
+			for (String adapterNameWithPossibleConfigurationName : adapters) {
+				int slash = adapterNameWithPossibleConfigurationName.indexOf("/");
+				String adapterName;
+				if (slash > -1) {
+					adapterName = adapterNameWithPossibleConfigurationName.substring(slash + 1);
+					String configurationName = adapterNameWithPossibleConfigurationName.substring(0, slash);
+					builder.addHeader(BusMessageUtils.HEADER_CONFIGURATION_NAME_KEY, configurationName);
+				} else {
+					adapterName = adapterNameWithPossibleConfigurationName;
+				}
+				builder.addHeader(BusMessageUtils.HEADER_ADAPTER_NAME_KEY, adapterName);
+				callAsyncGateway(builder);
+			}
+		}
+
+		return ResponseEntity.status(HttpStatus.ACCEPTED).build(); //PUT defaults to no content
+	}
+
+	@RolesAllowed({"IbisDataAdmin", "IbisAdmin", "IbisTester"})
+	@Relation("adapter")
+	@Description("start/stop an adapter")
+	@PutMapping(value = "/configurations/{configuration}/adapters/{adapter}", produces = MediaType.APPLICATION_JSON_VALUE, consumes = MediaType.APPLICATION_JSON_VALUE)
+	public ResponseEntity<?> updateAdapter(@PathVariable("configuration") String configuration, @PathVariable("adapter") String adapter, @RequestBody Map<String, Object> json) {
+		Object value = json.get("action");
+		if (value instanceof String) {
+			IbisAction action = null;
+			if (value.equals("stop")) {action = IbisAction.STOPADAPTER;}
+			if (value.equals("start")) {action = IbisAction.STARTADAPTER;}
+			if (action == null) {
+				throw new ApiException("no or unknown action provided", HttpStatus.BAD_REQUEST);
+			}
+
+			RequestMessageBuilder builder = RequestMessageBuilder.create(this, BusTopic.IBISACTION);
+			builder.addHeader("action", action.name());
+			builder.addHeader(BusMessageUtils.HEADER_CONFIGURATION_NAME_KEY, configuration);
+			builder.addHeader(BusMessageUtils.HEADER_ADAPTER_NAME_KEY, adapter);
+			callAsyncGateway(builder);
+			return ResponseEntity.status(HttpStatus.ACCEPTED).body("{\"status\":\"ok\"}");
+		}
+
+		return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
+	}
+
+	@RolesAllowed({"IbisDataAdmin", "IbisAdmin", "IbisTester"})
+	@Relation("adapter")
+	@Description("start/stop an adapter receivers")
+	@PutMapping(value = "/configurations/{configuration}/adapters/{adapter}/receivers/{receiver}", produces = MediaType.APPLICATION_JSON_VALUE, consumes = MediaType.APPLICATION_JSON_VALUE)
+	public ResponseEntity<?> updateReceiver(@PathVariable("configuration") String configuration, @PathVariable("adapter") String adapter, @PathVariable("receiver") String receiver, @RequestBody Map<String, Object> json) {
+		Object value = json.get("action");
+		if (value instanceof String) {
+			IbisAction action = null;
+			if (value.equals("stop")) {action = IbisAction.STOPRECEIVER;} else if (value.equals("start")) {
```

Comment:
These kind of code lines does not seem to comply with the standards we've put in. 
We've added `.editorconfig` files, which the IDE should pick up and use to format the code properly.

Reply:
Maybe a copy-paste issue, or intellij doesnt automatically format code?
I haven't changed my editor to not look at it so it would be weird that the default behaviour doesnt format for me

Reply:
Maybe you can correct this class, since this is a bit messed up.

Reply:
Reformatting didnt seem to work, I had to manually add correct line breaks. Maybe we should look into this at a later point in time

Reply:
Sure. But also feel free to improve it, even if you have to add line breaks. Or leave it, if you don't feel comfortable.